libs/backend/src/exporter.test.ts

```diff
@@ -148,14 +148,14 @@ test('exportDataToUsbDrive happy path', async () => {
   usbDrive.status
     .expectCallWith()
     .resolves({ status: 'mounted', mountPoint: tmpDir });
+  usbDrive.sync.expectCallWith().resolves();
```

Comment:
do these cases need an `assertComplete()` at the end to be equivalent to the previous code?

Reply:
Ah thank you, I just assumed it was already there